packages/gatsby-theme-newrelic/src/components/SwiftTypeSearch.js

```diff
@@ -68,6 +71,11 @@ const SwiftTypeSearch = ({ className }) => {
                   searchAsYouType
                   debounceLength={500}
                   inputView={InputView}
+                  onSubmit={(searchTerm) => {
+                    let queryString = qs.parse(location.search);
+                    queryString.q = searchTerm;
+                    navigate('/' + qs.stringify(queryString));
```

Comment:
Does this mean that updating the search term will always route the user back to the home page?